packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -398,6 +398,16 @@ class MethodChannelCamera extends CameraPlatform {
     );
   }
 
+  @override
+  Future<Size?> setCaptureMode(int cameraId, CaptureMode mode) =>
+      _channel.invokeMethod<Size>(
+        'setCaptureMode',
+        <String, dynamic>{
+          'cameraId': cameraId,
+          'mode': serializeCaptureMode(mode),
```

Comment:
nit:

Instead of using `serializeCaptureMode(mode)`, you should be able to use `mode.name` for the same value.